lib/elixir/lib/task/supervised.ex

```diff
@@ -168,6 +169,7 @@ defmodule Task.Supervised do
       reducer: reducer,
       monitor_pid: monitor_pid,
       monitor_ref: monitor_ref,
+      ordered: ordered?,
```

Comment:
I don't think we do that in any of the other APIs, so I wouldn't start it here. I should ship it as is.